live-build/misc/upgrade-scripts/verify-impl

```diff
@@ -26,6 +26,25 @@ function usage() {
 	exit 2
 }
 
+function get_container_name() {
+	return basename $(df -k / | tail -1 | awk '{print $1}')
+}
+
+function cleanup() {
+	local rc="$?"
+
+	#
+	# Persist container logs by saving to the container's host.
+	# Delete and create verify log destination directory to save only one copy of log to limit
+	# space usage.
+	#
+	rm -rf "${UPDATE_VERIFY_LOG_DIR:?}"
+	mkdir -p "${UPDATE_VERIFY_LOG_DIR}/${get_container_name}"
```

Comment:
Now that we're storing the logs in a per-container directory, how will these directories get cleaned up? Will there be logic in the app-gate that does this? or should we do this in the upgrade-scripts (e.g. perhaps when the container is destroyed)?

Reply:
I think I found the answer to my question in the comment above. We "cleanup" the logs for old runs, by removing the entire directory (including any old logs) each time we run verification. That seems reasonable to me.